tsdb/tsm1/config.go

```diff
@@ -79,10 +75,12 @@ type CompactionConfig struct {
 	MaxConcurrent int `toml:"max-concurrent"`
 }
 
+// Default Cache configuration values.
 const (
-	DefaultCacheMaxMemorySize             = 1024 * 1024 * 1024 // 1GB
-	DefaultCacheSnapshotMemorySize        = 25 * 1024 * 1024   // 25MB
-	DefaultCacheSnapshotWriteColdDuration = time.Duration(10 * time.Minute)
+	DefaultCacheMaxMemorySize             = toml.Size(1024 << 20)           // 1GB
```

Comment:
Minor nit but why not `1 << 30`?

Reply:
Just seemed clearer to use the same base (MB) for all the constants).